views/fanChart/SettingsOptions.js

```diff
@@ -108,8 +108,8 @@ SettingsOptions.SettingsOptionsObject = class SettingsOptionsObject {
             data.viewClassName +
             '.cancelSettings();">'+ SVGbtnCLOSE +'</A></span>' +
             this.createULelements(data) +
-            '<br />    <div align="center">      <div id="status"></div>      <button id="saveSettingsChanges" class="saveButton">Save changes (all tabs)</button>';
-        ("</div>");
+            '<br />    <div align="center">      <div id="status"></div>      <button id="saveSettingsChanges" class="saveButton">Save changes (all tabs)</button>' +
+        "</div>";
```

Comment:
This change is not enough. You need another `</div>  here to close `<div id=settingsDIV>`

Reply:
Fixed

Reply:
DONE !